providers/hubspot/webhook_test.go

```diff
@@ -66,3 +67,107 @@ func TestWebhook(t *testing.T) {
 	_, err = conn.ExtractObjectNameFromWebhookMessage(emptyObjectMessage)
 	assert.ErrorContains(t, err, "webhook is not supported for the object ''")
 }
+
+//nolint:funlen
+func TestExtractEventTypeFromWebhookMessage(t *testing.T) {
+	t.Parallel()
+
+	createMessage := &WebhookMessage{
+		AppId:            1,
+		EventId:          1,
+		SubscriptionId:   1,
+		PortalId:         1,
+		OccurredAt:       1,
+		SubscriptionType: "contact.creation",
+		AttemptNumber:    1,
+		ObjectId:         1,
+		ChangeSource:     "CRM",
+		PropertyName:     "message",
+		PropertyValue:    "sample-value",
+	}
+
+	conn := &Connector{}
+
+	evtTypeCreate, err := conn.ExtractEventTypeFromWebhookMessage(createMessage)
+	if err != nil {
+		t.Fatalf("error extracting object name from webhook message: %s", err)
+	}
+
+	assert.Equal(t, evtTypeCreate, common.WebhookEventTypeCreate, "object type should be parsedCorrectly")
```

Comment:
Please update the message to "object type should be create", same for the other ones in this file.